index.html

```diff
@@ -78,7 +80,7 @@
 
   <div id="map"></div>
 
-  <script data-spreadsheet="0Agcr__L1I1PDdEpoMnhxR0RHdkFsWlFtNTlEZlltR0E" src="js/script.js"></script>
+  <script data-spreadsheet="0Agcr__L1I1PDdEpoMnhxR0RHdkFsWlFtNTlEZlltR0E" data-map-args="actionkit=peoples-climate-plan-actions|break-free-world&controlshift=campaigns.gofossilfree.org:fossil-fuel-divestment-religious-institutions|campaigns.350.org:climate-impacts" src="js/script.js"></script>

```

Comment:
Let's leave the default map's layers as-is for now, i.e. remove this data-map-args addition.